application-ckeditor-ui/src/main/resources/CKEditor/MacroService.xml

```diff
@@ -30,9 +30,10 @@
   <creationDate>1458047924000</creationDate>
   <parent>CKEditor.WebHome</parent>
   <author>xwiki:XWiki.Admin</author>
+  <originalMetadataAuthor>XWiki.Admin</originalMetadataAuthor>
```

Comment:
I hope this won't cause issues when installed on an older versions of XWiki.

Reply:
It won't but it's still a mistake, I was not planning to commit that, fixing.